packages/forms/src/enhanced/useFormikEnhanced.ts

```diff
@@ -89,6 +97,9 @@ export function useFormikEnhanced<TValues extends FormikValues, TResponse>({
         }),
   }) as FormikContextTypeEnhanced<TValues, TResponse>;
 
+  // Attach children to the returned context for React 18 compatibility
+  formik.children = children;
+
```

Comment:
@SiriusCrain formik is basically an object here, not a component. Does React 18 require objects from hooks to have children fields?